types/parse/index.d.ts

```diff
@@ -608,6 +608,7 @@ declare namespace Parse {
         constructor(objectClass: string);
         constructor(objectClass: new (...args: any[]) => T);
 
+        static and<U extends Object>(arg0: Query<U>, arg1: Query<U>): Query<U>;
```

Comment:
Should accept a variable number of arguments
static and<U extends Object>(...var_args: Query<U>[]): Query<U>;